ansible/roles/uan_hardening/tasks/disable_ssh_out_nmn_to_nmn_lb.yml

```diff
@@ -0,0 +1,58 @@
+#
```

Comment:
Was this file included by mistake? There is a similarly named one that is included in the main.yml tasks.